ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -100,8 +100,10 @@ platform:
   baremetal:
     libvirtURI: >-
       qemu+ssh://root@${AUX_HOST}:$(sed 's/^[%]\?\([0-9]*\)[%]\?$/\1/' < "${CLUSTER_PROFILE_DIR}/provisioning-host-ssh-port-${architecture}")/system?keyfile=${CLUSTER_PROFILE_DIR}/ssh-key&no_verify=1&no_tty=1
-    apiVIP: $(yq ".api_vip" "${SHARED_DIR}/vips.yaml")
-    ingressVIP: $(yq ".ingress_vip" "${SHARED_DIR}/vips.yaml")
+    apiVIPs:
```

Comment:
Why do we need this if there is a new step that creates a patch for the install-config (used a few lines below)?

Reply:
I will revert and test, but I remember when running w/o touching this part, vips did not look good on install-config